### DISCUSSION TITLE
FR: Trivia available on its windows.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17459] [Feature requests]
I think it's important to credit where credit is due, and have this available on display. Gives more incentive to people to work on new objects aswell. Currently credits are mostly only available on the Object Selection menu and even in here it's difficult to find all stuff from a specific author unless they have put their username in the name or parkobj file name (not even ID).

Currently our object creators are still crediting themselves in the object names, which causes unnecessary clutter.

I was thinking of the following additions:
- Make the object selection's search engine able to search for the "Author" field aswell. 
- Add the Author name of the object in the right bottom corner of the window in the case of a rides and scenery items.
- Add the name and "by `<author>`" to a pop up text (tooltip) when hovering over path objects or terrain objects (similar to what you see below).
![image](https://user-images.githubusercontent.com/1516893/175991884-1d820020-700b-4eca-9dfa-610a2db8956a.png)
- For ride music see #17385

**Comment by ZehMatt** [comment]
Adding the author to the tooltips seems a bit too much in my opinion

**Reply by karst** [reply]
Yea i think so too

**Comment by J-PIE-314** [comment]
For scenery, the author names are now displayed during scenario object selection and in the various in-game locations when debugging tools are enabled. The names of musical artists are now displayed by default in the music selection tabs.
